pkg/federation/fed_single_cluster_install.go

```diff
@@ -46,7 +46,7 @@ func TestSingleClusterFed(t *testing.T) {
 	t.Run("federation_single_cluster_install", func(t *testing.T) {
 		defer util.RecoverPanic(t)
 		util.Log.Info("Test federation install in a single cluster")
-		util.Log.Info("Reference: https://github.com/maistra/istio/blob/maistra-2.1/pkg/servicemesh/federation/example/config-poc/install.sh")
+		util.Log.Info("Reference: https://github.com/maistra/istio/blob/maistra-2.4/samples/federation/base/install.sh")
```

Comment:
I think we should create a maistra-2.4 branch for this repo and then you can change your PR merge into maistra-2.4 branch

Reply:
It is good to have the 2.3 for the 2.3 branch

Reply:
Agree with @pbajjuri20 I'll make the changes for this log to reference the 2.3 doc and them we can create a new branch from this to have a new main 2.4 branch. Agree?

Reply:
Change made, after this merge will create a new branch 2.4 to track the new OSSM branch there